python/libcuopt/CMakeLists.txt

```diff
@@ -74,7 +74,9 @@ target_link_libraries(cuopt_cli PRIVATE
 set(rpaths
   "$ORIGIN/../lib64"
   "$ORIGIN/../../rapids_logger/lib64"
+  "$ORIGIN/../../nvidia/cu12/lib"
```

Comment:
We probably need to make the CUDA 12 / CUDA 13 lines conditional for only the relevant CUDA versions.